tests/io/test_crosssection.py

```diff
@@ -157,16 +156,16 @@ def test_create_a_zwrivercrsdef_with_wrong_list_lengths(self):
 
     def test_create_a_zwrivercrsdef_without_frictionspec(self):
         csdefid = "Prof1"
-        with pytest.raises(ValidationError) as error:
-            _ = ZWRiverCrsDef(
-                id=csdefid,
-                numlevels=2,
-                levels=[-2, 3],
-                flowwidths=[11, 44],
-            )
-        expected_message = f"Cross section is missing any friction specification."
-
-        assert expected_message in str(error.value)
+        crsdef = ZWRiverCrsDef(
+            id=csdefid,
+            numlevels=2,
+            levels=[-2, 3],
+            flowwidths=[11, 44],
+        )
+        assert crsdef.id == csdefid
+        assert crsdef.numlevels == 2
+        assert crsdef.levels == [-2, 3]
+        assert crsdef.flowwidths == [11, 44]
```

Comment:
Since validation passes, maybe also test for the other properties? 
Also, since the missing of the friction is now not a "special" case anymore (no if-statement to check for this situation), you normally don't test for this. But since we already have a test in place, I would leave it :)

Reply:
I've added testing on friction list fields to be None (because that's what this test is about.
all the other optional fields I've kept out of this test.